test/Extensions/TesterAzureUtils/Streaming/AQSubscriptionObserverWithImplicitSubscribingTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+using Orleans;
+using Orleans.Configuration;
+using Orleans.Hosting;
+using Orleans.Providers.Streams.AzureQueue;
+using Orleans.Streams;
+using Orleans.TestingHost;
+using Tester.StreamingTests.ProgrammaticSubscribeTests;
+using TestExtensions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Tester.AzureUtils.Streaming
+{
+    [TestCategory("Functional")]
+    public class AQSubscriptionObserverWithImplicitSubscribingTests : SubscriptionObserverWithImplicitSubscribingTestRunner, IClassFixture<AQSubscriptionObserverWithImplicitSubscribingTests.Fixture>
```

Comment:
This PR introduces test for SMS, and AQ, but not EH.  Should we also add to EH for more complete coverage?